app/services/ReviewService.scala

```diff
@@ -120,6 +117,15 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
         Future.successful(Left(errors))
     }
 
+  private def validateProcess(ap: ApprovalProcess, info: ApprovalProcessStatusChange) = {
+    ap.process
+      .validate[Process]
+      .fold(
```

Comment:
You can define the return type for the fold method before the open parentheses as below

fold[RequestOutcome[AuditInfo]](
        _ => Left(BadRequestError),
        process => Right(AuditInfo(info.userId, ap, process))
      )

Reply:
Have changed method to below which I think looks neater
`  private def validateProcess(ap: ApprovalProcess, info: ApprovalProcessStatusChange): RequestOutcome[AuditInfo] =
    ap.process
      .validate[Process]
      .fold(
        _ => Left(BadRequestError),
        process => Right(AuditInfo(info.userId, ap, process))
      )
`